htdocs/mrp/class/mo.class.php

```diff
@@ -706,7 +706,7 @@ public function updateProduction(User $user, $notrigger = true)
 							if ($moline->qty <= 0) {
 								$error++;
 								$this->error = "BadValueForquantityToConsume";
-								break;
+								continue;
```

Comment:
We have the $error++, so we should return an error code with the goal to cancel everything.
We want nothing happen if error. The the continue should be useless.
I thin the error is later when we make a commit instead of a rollback when the return code is < 0